.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: monthly
+    time: "03:00"
+  commit-message:
+    prefix: '[bot]'
+  labels:
+    - "dependencies"
+  open-pull-requests-limit: 0
```

Comment:
> This option has no impact on security updates, which have a separate, internal limit of ten open pull requests.
\- [Source](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#open-pull-requests-limit)

So this option is basically saying that dependabot will only create PRs for security updates when set to zero.